tutorials/02_uploading_program.ipynb

```diff
@@ -19,7 +19,7 @@
   {
    "cell_type": "markdown",
    "source": [
-    "Here we provide an overview on how to construct and upload a runtime program. A runtime program is a piece of Python code that lives in the cloud and can be invoked by other authorized users. Runtime programs are **private** by default, but the program visibility can be changed during upload or at a later time."
+    "Here we provide an overview on how to construct and upload a runtime program, available to premium users. A runtime program is a piece of Python code that lives in the cloud and can be invoked by other authorized users. Runtime programs are **private** by default, but the program visibility can be changed during upload or at a later time."
```

Comment:
Line 14 can probably also mention it's now open to premium users, so people don't close the tutorial immediately after seeing it lol